seed/csharp-sdk/examples/src/SeedExamples/Service/ServiceClient.cs

```diff
@@ -55,14 +55,14 @@ public async Task<string> CreateMovieAsync(Movie request)
     public async Task<object> GetMetadataAsync(GetMetadataRequest request)
     {
         var _query = new Dictionary<string, object>() { };
+        _query["tag"] = request
+            .Tag.Where(_value => _value != null)
+            .Select(_value => _value)
+            .ToList();
```

Comment:
@amckinney is the idea here that we are filtering out null values?

Reply:
I see

Reply:
Exactly - otherwise there are certain value formats that could throw exceptions like the one below:

```csharp
        // Without the .Where(...), the _value.Value access is unsafe.
        _query["maybeOperand"] = request
            .MaybeOperand.Where(_value => _value != null)
            .Select(_value => JsonSerializer.Serialize(_value.Value))
            .ToList();
```

I imagine we'll handle this in a separate component responsible for query parameter serialization like `qs` once we actually support `deepObject` query parameters.